phantom_token.c

```diff
@@ -286,47 +290,53 @@ static ngx_int_t handler(ngx_http_request_t *request)
 
                 return NGX_OK;
             }
-            else if (module_context->status == NGX_HTTP_NO_CONTENT)
+            else if (module_context->status == NGX_HTTP_NO_CONTENT || module_context->status == NGX_HTTP_UNAUTHORIZED)
             {
                 return set_www_authenticate_header(request, module_location_config, NULL);
             }
             else if (module_context->status == NGX_HTTP_SERVICE_UNAVAILABLE)
```

Comment:
I made some updates to logging for cases where an unexpected error occurs. For expected 401s due to missing, invalid or expired tokens we want the experience for existing customers to remain the same.